composer/airflow_1_samples/requirements.txt

```diff
@@ -1,5 +1,7 @@
-apache-airflow[gcp]==1.10.14
-kubernetes==18.20.0
+apache-airflow[gcp]==1.10.15
+apache-airflow-backport-providers-google==2021.3.3
+apache-airflow-backport-providers-cncf-kubernetes==2021.3.3
+kubernetes==11.0.0
```

Comment:
Why has the kubernetes version dropped from >18 to 11?

Reply:
It is the last version that's appropriate for these Airflow 1 samples - this is for long term support of a now unsupported version of Airflow (that we still support in Composer)